src/lib/routes/admin-api/api-token.ts

```diff
@@ -168,6 +171,26 @@ export class ApiTokenController extends Controller {
             ],
         });
 
+        this.route({
+            method: 'get',
+            path: '/:name',
+            handler: this.getApiTokensByName,
+            permission: [ADMIN, READ_CLIENT_API_TOKEN, READ_FRONTEND_API_TOKEN],
+            middleware: [
+                openApiService.validPath({
+                    tags: ['API tokens'],
+                    operationId: 'getApiTokensByName',
+                    summary: 'Get API tokens by name',
+                    description:
+                        'Retrieves all API tokens that match a given token name. Token names are not unique so 0, 1 or many tokens can be returned.',
```

Comment:
Small rewording suggestion. I think this is a little bit clearer, but I'll leave it up to you. 

```suggestion
                        'Retrieves all API tokens that match a given token name. Because token names are not unique, this endpoint will always return a list. If no tokens with the provided name exist, the list will be empty. Otherwise, it will contain all the tokens with the given name. 
```

If you want to keep your wording, you should add a comma between "unique" and "so": 
> Token names are not unique, so 0, 1 or many tokens can be returned.'